queries.sql

```diff
@@ -1,3 +1,68 @@
 /*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
-SELECT * from animals WHERE name = 'Luna';
+START TRANSACTION;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+START TRANSACTION;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] Great job on your progress in writing your query to delete all records from the animal's table. 🚀 However, there's an important step that's missing, which, when added, would enhance your code and ensure its robustness.

     - After the `ROLLBACK`, kindly consider querying the database one more time to verify that the rollback changes have been successfully undone in the database. This step provides you with peace of mind and a more foolproof process for managing your data. 🧐

Doing so ensures that your database remains in the expected state after executing the rollback, helping to prevent unexpected data inconsistencies. 

Happy coding! 🌟😊